src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -3055,13 +3055,13 @@ void addCloneMethod()
                 }
             }
 
-            ImmutableArray<PropertySymbol> addProperties(ImmutableArray<ParameterSymbol> recordParameters)
+            ImmutableArray<PropertySymbol> addProperties(Binder binder, ImmutableArray<ParameterSymbol> recordParameters)
```

Comment:
>Binder binder,  [](start = 57, length = 15)

This doesn't feel right. We didn't need the binder before and I don't see why we would need it now.